pkg/client/v1/nimbleos/client_credential.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Hewlett Packard Enterprise Development LP
+
+package nimbleos
+
+
+// ClientCredential - Credential that this device will trust.
+// Export ClientCredentialFields for advance operations like search filter etc.
+var ClientCredentialFields *ClientCredential
+
+func init(){
+ IDfield:= "id"
```

Comment:
Why single-character indentation?

Reply:
please use "go fmt" , formatting is off.